### DISCUSSION TITLE
发送的消息没有恢复，并且在终端提时(host='api.gradio.app', port=443): Read timed out

### DISCUSSION THREAD

**Discussion by caoxiaogang96** [GaiZhenbiao/ChuanhuChatGPT#525] [Q&A]
![屏幕截图 2023-04-03 162113](https://user-images.githubusercontent.com/53939014/229453471-3dc64eb4-b653-4457-8189-2208fc9c2e9a.png)
![屏幕截图 2023-04-03 162307](https://user-images.githubusercontent.com/53939014/229453486-513f76b0-98b0-4e31-8f9e-41a2c7a21dcf.png)

**Comment by GaiZhenbiao** [comment]
现已修复。这是因为OpenAI那边，这个接口挂了。
